spec/example_app/app/assets/stylesheets/administrate/customize.scss

```diff
@@ -0,0 +1,4 @@
+.custom-administrate-css-hidden {
```

Comment:
I thought sass treated underscores as partials.  I'm not sure what that really means in this context, but I usually read it as top-level vs. included.

That said, I've never really noticed any difference in behavior between the two, but I'm sure somebody will correct me on that.